hangman.html

```diff
@@ -0,0 +1,35 @@
+<html>
+<head>
+    <title>Hangman</title>
+    <link rel="stylesheet" href="hangman.css">
+    <script src="hangman.js" type="text/javascript"></script>
+</head>
+<body>
+    <div id="intro">
+        <h1>Celebrity Hangman</h1>
+    </div>
+        <div id="game-start">
+            <form id="input">
+            <p>Type a letter:
+                <input type="text" id="letter-input" autofocus>
+                <button id="submit">Submit</button>
+            </p>
+            </form>
+        </div>
+    <div class="info">
+    <h3 id="selected-word"></h3>
+    </div>
+    <div class="info">
+        <p id="remaining"></p>
+    </div>
+    <div class="info">
+    <p id="guess-letters"></p>
+    </div>
+    <div class="info">
+    <button id="reset" onclick="window.location()">Start New Game</button>
+    </div>
```

Comment:
I'm not completely sure why the onClick function was window.location.. I received an error. what were you expecting to happen on the click of your "start new game" button?